grakn/concept/thing/attribute.py

```diff
@@ -27,6 +26,9 @@
 
 class Attribute(Thing):
 
+    def get_value_type(self):
+        return self.get_type().get_value_type()
```

Comment:
hmmm i don't think we should expose this here - the users themselves can get the value type via the type. we're generally not providing shorthands unless it has overwhemling benefits/usage.